test/tests/functional/pbs_qstat.py

```diff
@@ -117,9 +117,10 @@ def validate_el(self, attr_dict, user):
                                      " does not match expected value" % each)
                 self.logger.info("Server attribute \"%s\""
                                  " is validated successfully" % each)
-        value = { k : qstat_o[k] for k in set(qstat_o) - set(attr_dict) }
+        value = [k for k in qstat_o.keys() if k not in attr_dict.keys()]
```

Comment:
Just as a note, you don't need to use .keys().  When you say x in dict, you're matching a key.  This will be slightly slower since you have to create a list when you don't really need to, but not overly so.  Change it if you want.